debian/ceph-exporter.postinst

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+# vim: set noet ts=8:
+# postinst script for ceph-exporter
+#
+# see: dh_installdeb(1)
+
+set -e
+
+# summary of how this script can be called:
+#
+# 	postinst configure <most-recently-configured-version>
+# 	old-postinst abort-upgrade <new-version>
+# 	conflictor's-postinst abort-remove in-favour <package> <new-version>
+# 	postinst abort-remove
+# 	deconfigured's-postinst abort-deconfigure in-favour <failed-install-package> <version> [<removing conflicting-package> <version>]
+#
+
+# for details, see http://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+[ -f "/etc/default/ceph" ] && . /etc/default/ceph
+[ -z "$SERVER_USER" ] && SERVER_USER=ceph
+[ -z "$SERVER_GROUP" ] && SERVER_GROUP=ceph
+
+case "$1" in
+    configure)
+	[ -x /sbin/start ] && start ceph-exporter-all || :
```

Comment:
why do we need to check for `/sbin/start`? i guess you are copying from other `.postinst` scripts. but we just forgot to update them. i created #55437 to address this.

Reply:
As you pointed out, I used other `.postinst` script as a base file.
I fixed it in the same way as https://github.com/ceph/ceph/pull/55437.